src/discover/discover.ml

```diff
@@ -177,12 +178,27 @@ let brew_libffi_version flags =
   | { Commands.stdout } ->
     String.trim stdout
 
+let msys_env_args () =
+  let pcp = Sys.getenv_opt "PKG_CONFIG_PATH" in
```

Comment:
We still support OCaml 4.03. `Sys.getenv_opt` and `Option` are not available.

Reply:
Makes sense. Will fix, and will see about installing a OCaml 4.03 compiler on Linux to regression test.